balancer/grpclb/grpclb_test.go

```diff
@@ -658,7 +658,7 @@ func (s) TestDropRequest(t *testing.T) {
 		if _, err := testC.EmptyCall(ctx, &testpb.Empty{}, grpc.WaitForReady(true), grpc.Peer(&p)); err != nil {
 			t.Errorf("%v.EmptyCall(_, _) = _, %v, want _, <nil>", testC, err)
```

Comment:
I think a better fix may be to make this a `t.Fatal` or `t.Errorf` followed by `continue`.

@menghanl

Reply:
`t.Fatal` sounds good

Reply:
updated. @dfawley